custom/icds_reports/management/commands/sql_scripts/fix_past_data_part_2.sql

```diff
@@ -0,0 +1,70 @@
+DROP TABLE IF EXISTS temp_chm;
+CREATE UNLOGGED TABLE temp_chm (LIKE agg_child_health);
+SELECT create_distributed_table('temp_chm', 'supervisor_id');
+INSERT INTO "temp_chm" (
+  awc_id, supervisor_id, month, age_tranche, caste,
+  wasting_moderate, wasting_severe, wasting_normal, stunting_moderate, stunting_severe,
+  stunting_normal, height_measured_in_month, weighed_and_height_measured_in_month
+) (
+  SELECT
+      chm.awc_id, chm.supervisor_id, chm.month, chm.age_tranche, chm.caste,
+      SUM(
+        CASE WHEN chm.current_month_wasting = 'moderate' THEN 1 ELSE 0 END
+      ) as wasting_moderate,
+      SUM(
+        CASE WHEN chm.current_month_wasting = 'severe' THEN 1 ELSE 0 END
+      ) as wasting_severe,
+      SUM(
+        CASE WHEN chm.current_month_wasting = 'normal' THEN 1 ELSE 0 END
+      ) as wasting_normal,
+      SUM(
+        CASE WHEN chm.current_month_stunting = 'moderate' THEN 1 ELSE 0 END
+      ) as stunting_moderate,
+      SUM(
+        CASE WHEN chm.current_month_stunting = 'severe' THEN 1 ELSE 0 END
+      ) as stunting_severe,
+      SUM(
+        CASE WHEN chm.current_month_stunting = 'normal' THEN 1 ELSE 0 END
+      ) as stunting_normal,
+      SUM(chm.height_measured_in_month) as height_measured_in_month,
+      SUM(
+        CASE WHEN chm.nutrition_status_weighed = 1
+        AND chm.height_measured_in_month = 1 THEN 1 ELSE 0 END
+      ) as weighed_and_height_measured_in_month
+    FROM
+      "child_health_monthly" chm
+    WHERE
+      chm.month = '{start_date}'
+    GROUP BY
+      chm.awc_id,
+      chm.supervisor_id,
+      chm.month,
+      chm.age_tranche,
```

Comment:
The agg level 5 group is wrong here. it also include other fields as well https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/aggregation_helpers/distributed/agg_child_health.py#L185

Reply:
updated!